### DISCUSSION TITLE
Amazon SC_VAT_TAX_REPORT_INVOICE

### DISCUSSION THREAD

**Discussion by michelexport** [amzn/selling-partner-api-models#3330] [General]
Hi all,
we are pulling SC_VAT_TAX_REPORT using amazon sp-api.
We need to download datas on a daily basis, amazon customer assistance said that the report is available every 12hrs.
However, when we run our script, it seems that datas are available with a 3days delay.

Any of you can download this report daily?

Looking forward to a solution
![Capture](https://user-images.githubusercontent.com/103668476/168779167-cd0766f0-1c8e-48f1-a5e0-313797885137.JPG)
!

**Comment by Holdinar** [comment]
Hey, have you found out what the problem was?

**Reply by michelexport** [reply]
Hi there, yes, all solved thanks :)

**Reply by Philomath88** [reply]
So what was the solution @michelexport?

**Reply by michelexport** [reply]
@Philomath88 we had an issue with our script, all good now. Report via Api is available every 12hrs
